### PR TITLE
feat(components): skeleton

### PR DESCRIPTION
**Pull request by x8a**
Added a skeleton component so we can use it with Split tests.

### CODE REVIEW

File: packages/gatsby-theme-newrelic/src/components/Skeleton.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { css } from '@emotion/react';
+
+const Skeleton = () => (
+  <div
+    css={css`
+      width: 100px;
+      height: 100px;
+      animation: skeleton-loading 1s linear infinite alternate;
+      @keyframes skeleton-loading {
```
Comment (caylahamann) [inline packages/gatsby-theme-newrelic/src/components/Skeleton.js:10]: this is looking great, one thing that still remains is to account for dark mode. We'll want to add styles for when the dark mode class name is applied to the page. It should look something like this: 
https://github.com/newrelic/gatsby-theme-newrelic/blob/e92c4259148b0349968629029f5a3cbaa41dce7f/packages/gatsby-theme-newrelic/src/components/TextHighlight.js#L25-L27

File: demo/src/pages/index.js [hunk 2]
```diff
@@ -139,6 +140,9 @@ const IndexPage = () => {
             <li>Final list item</li>
           </ul>
           <h2>Search inputs</h2>
```
Comment (caylahamann) [inline demo/src/pages/index.js:142]: small thing, but I notice that in the demo this breaks up the `SearchInput` section. You're going to want to move this `h2` to be right next to the `SearchInput`

**Review by caylahamann — CHANGES_REQUESTED** [review]
Let me know if you have any questions about this feedback!!

**Review by caylahamann — APPROVED** [review]
This looks great!!! thanks for making those changes

**Comment by x8a** [comment]
@roadlittledawn, thanks for the notes! I added those 💃🏻.

**Pull request merged**